### DISCUSSION TITLE
自定义样式怎么在本地调试？

### DISCUSSION THREAD

**Discussion by YMC0182** [giscus/giscus#1406] [Q&A] [answered]
配置css链接必须是线上的吗？我引入本地文件不生效，怎么在本地调试样式？

**Comment by zhenyou-nide** [comment, marked as answer]
一个思路是: 
css 文件放在 public 下，`data-theme={location.origin + '/custom.css'}`

repo root 下添加 giscus.json ,
```json
{
  "origins": ["https://giscus.app"], // 这里可以写你生产的地址
  "originsRegex": ["http://localhost:[0-9]+"] // 这里把你本地环境的 origin 列入
}
```

详细示例：https://github.com/giscus/giscus/blob/main/giscus.json

**Comment by YMC0182** [comment]
在public下新建css文件giscus.css，引入http://localhost:1234/giscus.css

直接引入./discus。css 不生效
